Makefile

```diff
@@ -147,20 +138,9 @@ devnet-clean:
 	docker volume ls --filter name=ops-bedrock --format='{{.Name}}' | xargs -r docker volume rm
 .PHONY: devnet-clean
 
-devnet-legacy-clean:
```

Comment:
I guess we undo some options here, is it right? :D

Reply:
Yes, I'm sorry for my lack of explanation.
I think we can't supply devnet-legacy (ETH native) if we change the name of LEGACY_ERC20_ETH in our `op-chain-ops`. So I asked that Is it okay if we remove devnet-legacy to contracts team.